etc/configuration/default.json

```diff
@@ -633,7 +633,7 @@
     "angular_velocity_low_pass_factor": 0.2,
     "roll_pitch_low_pass_factor": 0.2,
     "gravitational_acceleration_threshold": 4.0,
-    "falling_angle_threshold": [0.5, 0.45],
+    "falling_angle_threshold": [0.5, 0.4],
```

Comment:
These are changed by #212 which is already approved for merge, you may need to adjust accordingly and potentially retest to make sure functionality remains as expected

Reply:
Done. It is changed and the motion is adapted to this parameter.